classes/controller/AdminController.php

```diff
@@ -2145,7 +2150,12 @@ protected function addPageHeaderToolBarModulesListButton()
     {
         $this->filterTabModuleList();
 
-        if (is_array($this->tab_modules_list['slider_list']) && count($this->tab_modules_list['slider_list'])) {
+        /**
+         * @TODO remove isInstalled when MBO module will be released
+         */
+        if (!Module::isInstalled('ps_mbo') &&
```

Comment:
We may do this on the module too, so this shouldn't be merged right away

Reply:
Same as previous comment, the only way I find is to use a javascript to hide links :/ 
If everything must be done in MBO module, should we see with @Alexisvallee?